SADXModManager/MainForm.cs

```diff
@@ -933,35 +929,35 @@ private void Save()
                 loaderini.Mods.Add((string)item.Tag);
             }
 
-           	loaderini.DebugConsole              = consoleCheckBox.Checked;
-			loaderini.DebugScreen               = screenCheckBox.Checked;
-			loaderini.DebugFile                 = fileCheckBox.Checked;
-			loaderini.DisableCDCheck            = disableCDCheckCheckBox.Checked;
-			loaderini.HorizontalResolution      = (int)horizontalResolution.Value;
-			loaderini.VerticalResolution        = (int)verticalResolution.Value;
-			loaderini.ForceAspectRatio          = forceAspectRatioCheckBox.Checked;
-			loaderini.ScaleHud                  = checkScaleHud.Checked;
-			loaderini.BackgroundFillMode        = comboBackgroundFill.SelectedIndex;
-			loaderini.FmvFillMode               = comboFmvFill.SelectedIndex;
-			loaderini.EnableVsync               = checkVsync.Checked;
-			loaderini.WindowedFullscreen        = windowedFullscreenCheckBox.Checked;
-			loaderini.AutoMipmap                = forceMipmappingCheckBox.Checked;
-			loaderini.TextureFilter             = forceTextureFilterCheckBox.Checked;
-			loaderini.PauseWhenInactive         = pauseWhenInactiveCheckBox.Checked;
-			loaderini.StretchFullscreen         = stretchFullscreenCheckBox.Checked;
-			loaderini.ScreenNum                 = screenNumComboBox.SelectedIndex;
-			loaderini.CustomWindowSize          = customWindowSizeCheckBox.Checked;
-			loaderini.WindowWidth               = (int)windowWidth.Value;
-			loaderini.WindowHeight              = (int)windowHeight.Value;
-			loaderini.MaintainWindowAspectRatio = maintainWindowAspectRatioCheckBox.Checked;
-			loaderini.ResizableWindow           = checkWindowResize.Checked;
-			loaderini.UpdateCheck               = checkUpdateStartup.Checked;
-			loaderini.ModUpdateCheck            = checkUpdateModsStartup.Checked;
-			loaderini.UpdateUnit                = (UpdateUnit)comboUpdateFrequency.SelectedIndex;
-			loaderini.UpdateFrequency           = (int)numericUpdateFrequency.Value;
-			loaderini.VoiceLanguage             = (int)comboVoiceLanguage.SelectedIndex;
-			loaderini.TextLanguage              = (int)comboTextLanguage.SelectedIndex;
-			loaderini.DisableBASS = checkBASS.Checked;
+               loaderini.DebugConsole              = consoleCheckBox.Checked;
```

Comment:
Extra indentation here.
```suggestion
            loaderini.DebugConsole              = consoleCheckBox.Checked;
```